src/test/java/com/godaddy/sonar/ruby/rubocop/RubocopSensorTest.java

```diff
@@ -81,16 +83,22 @@ public void shouldAnalyzeAndSetCorrectRubocopProblemsWhenReportFileDoesNotExists
     private InputFile inputFile(String relativePath, InputFile.Type type) {
 
         // generate the default input file by the relative path and type given
-        DefaultInputFile inputFile = new DefaultInputFile("modulekey", relativePath)
-                .setModuleBaseDir(moduleBaseDir.toPath())
-                .setLanguage("ruby")
-                .setType(type);
-
-        // set the corresponding file metadata and add to context file system
-        inputFile.initMetadata(new FileMetadata().readMetadata(inputFile.file(), Charsets.UTF_8));
-        context.fileSystem().add(inputFile);
-
-        return inputFile;
+        try {
+            DefaultInputFile inputFile = new TestInputFileBuilder("modulekey", relativePath)
+                    .setModuleBaseDir(moduleBaseDir.toPath())
+                    .setLanguage("ruby")
+                    .setType(type)
+                    .build();
+
+            System.out.println(moduleBaseDir.toString() + relativePath);
```

Comment:
Remove system logging and move debug logging to the logger if relevant. See[1]
[1] https://github.com/shakedlokits/ruby-sonar-plugin/blob/6.7-dev/src/main/java/com/godaddy/sonar/ruby/metricfu/MetricfuComplexitySensor.java#L70

Reply:
Was only relevant while updating the test, I forgot to remove it with the other debugging statements. Removed.